xcode/Subconscious/Shared/Components/Common/SubtextView.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  SubtextView.swift
+//  Subconscious
+//
+//  Created by Gordon Brander on 4/18/23.
+//
+
+import SwiftUI
+
+struct SubtextView: View {
+    private static var renderer = SubtextAttributedStringRenderer()
+    var subtext: Subtext
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            ForEach(subtext.blocks, id: \.self) { block in
+                Text(Self.renderer.render(block.description))
```

Comment:
<img src="https://user-images.githubusercontent.com/5009316/234722683-baafa78b-ccc0-471f-a0ca-f43732236ccb.png" width=256>

We should add `.textSelection(.enabled)` to the `Text` or the `VStack` so that users can copy/share the text in blocks. While this lets you share a block, it seems there is no way to get actual region-based text selection without writing a custom view, see: https://stackoverflow.com/questions/58005434/how-do-i-allow-text-selection-on-a-text-label-in-swiftui

Reply:
Good catch.

Annoying that there is no easy way to support full-document select across text elements. I can accept that as a tradeoff for now, if it gets us to view-only transclusion faster.